docs/openapi/components/schemas/credentials/MillTestReportCredential.yml

```diff
@@ -729,6 +729,13 @@ properties:
                   $linkedData:
                     term: mo
                     "@id": https://w3id.org/traceability#mo
+                sn:
+                  title: Sn
+                  description: Tin percentage content.
+                  type: number
+                  $linkedData:
+                    term: sn
+                    "@id": https://w3id.org/traceability#sn
```

Comment:
https://w3id.org/traceability#sn does not actually resolve, please file an issue to make it resolve or replace it with a better term URL.

Reply:
Yes, good!
https://github.com/w3c-ccg/traceability-vocab/issues/846